src/main/java/org/folio/fqm/client/SimpleHttpClient.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.fqm.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@Service
+@FeignClient(name = "simple-http-client", url = ".")
```

Comment:
It took me soooo long to figure this part out :P The name is required, but it sets the base path for the client. After a lot of trial and error, I figured out that setting `url` to `"."` will override the base path with something that doesn't actually do anything, so that the path in `get()` actually works